pkg/cmd/server/origin/master.go

```diff
@@ -287,7 +287,7 @@ func (c *MasterConfig) buildHandlerChain() (func(apiHandler http.Handler, kc *ap
 
 			// these are all equivalent to the kube handler chain
 			///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
-			handler = serverhandlers.AuthorizationFilter(handler, c.Authorizer, c.AuthorizationAttributeBuilder, genericConfig.RequestContextMapper)
+			handler = apifilters.WithAuthorization(apiHandler, c.RequestContextMapper, c.Authorizer)
```

Comment:
I think you want `handler`, not `apiHandler`.

Reply:
Yikes.  Hopefully that's what failed test-cmd.